modules/persistent-storage-csi-google-cloud-file-create-sc.adoc

```diff
@@ -25,11 +25,13 @@ metadata:
   name: filestore-csi
 provisioner: filestore.csi.storage.gke.io
 parameters:
-  network: network-name <1>
+  connect-mode: PRIVATE_SERVICE_ACCESS <1>
+  network: network-name <2>
 allowVolumeExpansion: true
 volumeBindingMode: WaitForFirstConsumer
 --
-<1> Specify the name of the GCP virtual private cloud (VPC) network where Filestore instances should be created in.
+<1> For a shared VPC, use the `connect-node` parameter set to `PRIVATE_SERVICE_ACCESS`.
```

Comment:
I would mention this setting is optional. The default (DIRECT_PEERING) works for non shared VPC. You need to include this option if you are using a shared VPC.
https://github.com/kubernetes-sigs/gcp-filestore-csi-driver#supported-createvolume-parameters

As most deployments will not necessarily use a share VPC it may not be a good idea to show this option as default.

Reply:
Good point. Fixed.